src/ode_solver/explicit_ode_solver.cpp

```diff
@@ -96,6 +112,18 @@ void RungeKuttaODESolver<dim,real,n_rk_stages,MeshType>::step_in_time (real dt,
     }
     this->dg->solution = this->solution_update; // u_np1 = u_n + dt* sum(k_i * b_i)
 
+    // Apply limiter at every RK stage
+    if ( limiter_type != limiter_enum::none || apply_tvb == true) {
+        this->limiter->limit(this->dg->solution,
+            this->dg->dof_handler,
+            this->dg->fe_collection,
+            this->dg->volume_quadrature_collection,
+            this->dg->high_order_grid->fe_system.tensor_degree(),
+            this->dg->max_degree,
+            this->dg->oneD_fe_collection_1state,
+            this->dg->oneD_quadrature_collection);
```

Comment:
to reduce arguments; could you simply pass the `dg` object and access the others inside the `limit()` function ?

Reply:
Nah, I suggested we pass only what's needed. At the moment, we're passing DG around like a container for global variables. Limiters only need the solution. In the future, I'll probably have a GlobalSolution class.

Reply:
That would be quite nice to have a GlobalSolution class, I think it would simplify many parts of the code.